app/src/main/scala/org/alephium/explorer/persistence/queries/QuerySplitter.scala

```diff
@@ -23,22 +23,25 @@ import slick.dbio.DBIOAction
 import org.alephium.explorer.persistence.DBActionW
 import org.alephium.explorer.persistence.schema.CustomSetParameter.paramPlaceholder
 
-object QueryUtil {
+object QuerySplitter {
 
-  /** Maximum number of parameters allowed by Postgres per query.
+  /** Maximum number of parameters per query.
+    *
+    * @note This value cannot be > [[scala.Short.MaxValue]].
+    *       See issue <a href="https://github.com/alephium/explorer-backend/issues/160">#160</a>.
     *
     * TODO: This should be passed as a configuration instead of hardcoding it
     *       and should be passed as parameter to the function using this value.
     * */
-  val postgresDefaultParameterLimit: Short = Short.MaxValue
+  val maxParametersPerQuery: Short = 2000
```

Comment:
We could introduce `val maxRowPerQuery: Short = 200` to make the cache bounded by the number of rows, instead of number of parameters. `maxParametersPerQuery` would be `maxRowPerQuery * rowParams`

Reply:
- `val maxRowPerQuery: Short = 200` added [here](https://github.com/alephium/explorer-backend/pull/170/files#diff-c5205d5c5aadb4be1f66145f2ff3256aa1aa61730abd1acbd2e2af75c2bb201dR39)
- `maxRowPerQuery * rowParams` added [here](https://github.com/alephium/explorer-backend/pull/170/files#diff-c5205d5c5aadb4be1f66145f2ff3256aa1aa61730abd1acbd2e2af75c2bb201dR78)